scripts/dev/generate_epJSON_schema/idd_parser.py

```diff
@@ -110,15 +111,15 @@ def parse_idd(data):
         elif token == TOKEN_EXCLAMATION:
             eat_comment(data)
         elif token == TOKEN_GROUP:
-            next_token(data)
-            eat_comment(data)
+            current_group_name = parse_group(data).replace('\\group ', '')
         else:
             obj_name = parse_string(data)
             if obj_name is None or obj_name == "":
                 return root
             obj_data = parse_obj(data)
             root['properties'][obj_name] = {}
             root['properties'][obj_name]['patternProperties'] = {}
+            root['properties'][obj_name]['group'] = current_group_name
```

Comment:
I used the field name "group", but would be open to discussion if we want something better.